index.html

```diff
@@ -328,18 +327,18 @@ <h1 class="form__h1 poppins">
                 <ul class="form__list">
                     <li class="form__list1">
                         <input class="poppins form__list__style" name="name" type="text" id="Firstname"
-                            placeholder="Najash" maxlength="30" required />
+                            placeholder="Najash" maxlength="30" aria-label="User Name" required/>
                     </li>
                     <li>
                         <input class="poppins form__list__style" type="email" name="email" id="email"
-                            placeholder="email@something.com" required />
+                            placeholder="email@something.com" aria-label="Email" required />
                     </li>
                     <li class="form__list2">
                         <textarea class="poppins form__list__style" name="textarea__form" id="textarea__form" cols="30"
-                            rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                            rows="10" maxlength="500" aria-label="Enter Message" placeholder="Write your message here" required></textarea>
                     </li>
                     <li class="form__button__li">
-                        <button type="submit" class="form__button poppins">Get In Touch</button>
+                        <button type="submit" class="form__button poppins" aria-label="Get In Touch">Get In Touch</button>
                     </li>
```

Comment:
![color3](https://user-images.githubusercontent.com/93039359/187474254-8d939057-13e1-4d50-8e32-a3e594a2c3fd.png)
- [ ] Your button should have a color contrast of 4.5:1 ratio due to accessibility purpose. Please make it :+1: